app/src/lib/editors/linux.ts

```diff
@@ -18,23 +18,39 @@ interface ILinuxExternalEditor {
 const editors: ILinuxExternalEditor[] = [
   {
     name: 'Atom',
-    paths: ['/snap/bin/atom', '/usr/bin/atom'],
+    paths: [
+      '/snap/bin/atom', 
+      '/usr/bin/atom',
+      '/usr/bin/env atom'
```

Comment:
Have we tested this out? I feel like this is going to error because this string is expected to be a path, and having the additional parameter is not going to work...

Reply:
i've tested this so many times. this is how `unix` system works, and yea - you can try it yourself, just type `env <binaryname>` and it will start! (ofc untill this binary is exist into PATH. you can check path with `env` also, typing it into current terminal.)
and yes - this will fix nixos/nix issue that i've mentioned, i've also tested it localy.

Reply:
Sorry, not tested the approach but instead tested the app can handle this parameter. I fear it might blow up, but I also fear that it might skip over this scenario because this is doing a "path exists" check - defeating the intent of the change...

Reply:
ah, i haven't stumble across this issue. we always can create testing branch, merge it into and test it, i will help as i need it.

Reply:
I think I can figure out how to test this locally, but it's gonna take some focus time to dig into it. Perhaps later this week, otherwise definitely on the weekend...